src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -3,25 +3,69 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO.Compression;
+using System.IO;
 using System.Linq;
+using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
 using Amazon.ElasticBeanstalk;
 using Amazon.ElasticBeanstalk.Model;
 using AWS.Deploy.Common;
 using AWS.Deploy.Common.IO;
 using AWS.Deploy.Common.Recipes;
+using System.Text.Json.Serialization;
 
 namespace AWS.Deploy.Orchestration.ServiceHandlers
 {
     public interface IElasticBeanstalkHandler
     {
+        void SetupWindowsDeploymentManifest(Recommendation recommendation, string dotnetZipFilePath);
```

Comment:
Document new interface methods and the new class in this file.

Reply:
Done